editor/.eslintrc-lite.js

```diff
@@ -1,186 +1,3 @@
 module.exports = {
```

Comment:
do we still need this file for anything?

Reply:
I replaced now all of its occurences with references to `eslint-config-utopia`